library/ix-dev/community/overseerr/Chart.yaml

```
@@ -0,0 +1,23 @@
+name: overseerr
+description: Overseerr is a free and open source software application for managing requests for your media library.
+annotations:
+  title: Overseerr
+type: application
+version: 1.32.5
+apiVersion: v2
+kubeVersion: '>=1.16.0-0'
+maintainers:
+  - name: truenas
+    url: https://www.truenas.com/
```

```suggestion
    url: https://www.truenas.com/
    email: dev@ixsystems.com
```
Adding this as it's a new requirement 
https://github.com/truenas/charts/pull/1165